Tests/TestingApps/PurchaseTesterSwiftUI/Shared/Views/HomeView.swift

```diff
@@ -273,6 +273,8 @@ private struct CustomerInfoHeaderView: View {
                 #if targetEnvironment(macCatalyst) || os(macOS)
                 if #available(macCatalyst 16.0, *) {
                     OpenWindowButton()
+
+                    OpenProxyWindowButton()
```

Comment:
I'm wondering, should this button also exist in the configuration screen? In case we want to switch the proxy mode before we configure the SDK.

Reply:
Yeah good point! Once you open the window it remembers it and stays open every time you open the app, but technically there's no way to open it on first launch, so I'll do that.

Reply:
Hmm so the app will remember the proxy mode across launches? I would have guessed it should just use the `/status` endpoint to get the current mode and use that... I think modifying the proxy mode on app launch can be confusing IMO. Maybe I understood you incorrectly though.

Reply:
> Hmm so the app will remember the proxy mode across launches? 

No it doesn't, it'll refetch it. It just remembers that the window was open.

Reply:
> I'm wondering, should this button also exist in the configuration screen? In case we want to switch the proxy mode before we configure the SDK.

This is a bit of extra work so I'll punt for now. You can still open the window, then click "reconfigure" to change the mode before starting the SDK.